lte/gateway/python/magma/enodebd/data_models/data_model_parameters.py

```diff
@@ -85,6 +85,35 @@ class ParameterName():
     PERF_MGMT_USER = 'Perf mgmt username'
     PERF_MGMT_PASSWORD = 'Perf mgmt password'
 
+    # HO algorithm parameters
+    A1_THRESHOLD_RSRP = 'A1 threshold rsrp'
+    LTE_A1_THRESHOLD_RSRQ = 'Lte a1 threshold rsrq'
+    HYSTERESIS = 'Hysteresis'
+    TIME_TO_TRIGGER = 'Time to trigger'
+    A2_THRESHOLD_RSRP = 'A2 threshold rsrp'
+    LTE_A2_THRESHOLD_RSRQ = 'Lte a2 threshold rsrq'
+    LTE_A2_THRESHOLD_RSRP_IRAT_VOLTE = 'Lte a2 threshold rsrp irat volte'
+    LTE_A2_THRESHOLD_RSRQ_IRAT_VOLTE = 'Lte a2 threshold rsrq irat volte'
+    A3_OFFSET = 'a3 offset'
+    A3_OFFSET_ANR = 'a3 offset anr'
+    A4_THRESHOLD_RSRP = 'a4 threshold rsrp'
+    LTE_INTRA_A5_THRESHOLD_1_RSRP = 'lte intra a5 threshold1 rsrp'
+    LTE_INTRA_A5_THRESHOLD_2_RSRP = 'lte intra a5 threshold2 rsrp'
+    LTE_INTER_ANR_A5_THRESHOLD_1_RSRP = 'lte inter anra5 threshold1 rsrp'
+    LTE_INTER_ANR_A5_THRESHOLD_2_RSRP = 'lte inter anra5 threshold2 rsrp'
+    B2_THRESHOLD1_RSRP = 'b2 threshold1 rsrp'
+    B2_THRESHOLD2_RSRP = 'b2 threshold2 rsrp'
+    B2_GERAN_IRAT_THRESHOLD = 'b2 geran irat threshold'
+    QRXLEVMIN_SIB1 = 'qrxlevmin sib1'
+    QRXLEVMINOFFSET = 'qrxlevminoffset'
+    S_INTRASEARCH = 's intrasearch'
+    S_NONINTRASEARCH = 's nonintrasearch'
+    QRXLEVMIN_SIB3 = 'qrxlevmin sib3'
+    RESELECTION_PRIORITY = 'reselection priority'
+    THRESHSERVINGLOW = 'threshservinglow'
+    CIPHERING_ALGORITHM = 'ciphering algorithm'
+    INTEGRITY_ALGORITHM = 'integrity algorithm'
+
```

Comment:
prefix all of these with HO_
Also, many of these parameters are not self explanatory. Either add detailed commenting or improve param naming
Also, instead of using Sib1/Sib3, use cell selection/reselection terminology
Same for units used by each, db vs linear ...

Reply:
![image](https://user-images.githubusercontent.com/87301024/147403820-ba3dab16-bbea-4512-b7b9-712077851fe2.png)
![image](https://user-images.githubusercontent.com/87301024/147403831-0a650ee1-310b-4dae-bedf-a91cd5c89cd4.png)
Redefined and modified, thank you!